.github/workflows/masterbot.yml

```diff
@@ -164,26 +164,36 @@ jobs:
   prerelease:
     timeout-minutes: 40
     name: Pre-release
-    # TODO: Uncomment job dependencies when we enable new versioning process.
+    # TODO: Uncomment when we enable new versioning process.
+    # env:
+    #   NODE_AUTH_TOKEN: ${{ secrets.NPM_TOKEN }}
     # needs:
     #   - 'lint-check'
     #   - 'unit-test'
     #   - 'e2e-atomic-test'
     #   - 'e2e-quantic-test'
     runs-on: ubuntu-latest
-    environment: prerelease
+    environment: 'Prerelease'
     steps:
       - uses: actions/checkout@v3
         with:
           fetch-depth: 0
-      - uses: ./.github/actions/setup
+      - uses: actions/setup-node@v3
+        with:
+          # registry-url: 'https://registry.npmjs.org' # TODO: Uncomment when we enable new versioning process.
```

Comment:
See documentation for `registry-url`, which also explains why I'm adding `NODE_AUTH_TOKEN` to the whole job instead of one action:
```md
    # Optional registry to set up for auth. Will set the registry in a project level .npmrc and .yarnrc file, 
    # and set up auth to read in from env.NODE_AUTH_TOKEN.
    # Default: ''
    registry-url: ''
```